lib/trento/hosts.ex

```diff
@@ -136,6 +139,23 @@ defmodule Trento.Hosts do
     end
   end
 
+  @spec request_operation(atom(), String.t(), map()) :: {:ok, String.t()} | {:error, any}
+  def request_operation(:saptune_solution_apply, host_id, params) do
+    operation_id = UUID.uuid4()
+
+    with :ok <-
```

Comment:
Could we use a `case` instead of a single step with? It improves readability and allow to read the code better when a single operation is involved and the error handling is more clear